tidepods/write_pfs.py

```diff
@@ -68,21 +68,21 @@ def write_pfs(shp, template, date, pfs):
     pfs : str
           File path to pfs file.
 
-    '''
+    """
     date_str = str(date)
     acq_year = date_str[:4]
     plist = generate_pts(shp)
     formatted_pts = []
-    pid = 1
-    newline = '\n'
-    for p in plist:
-        point_text = ''.join(['[Point_', str(pid), ']', newline,
-                              'description = ', str(pid), newline,
-                              'y = ', str(p.coords[0][0]), newline,
-                              'x = ', str(p.coords[0][1]), newline,
-                              'EndSect // Point_', str(pid), newline])
-        formatted_pts.append(point_text)
-        pid += 1
+    POINT_FMT = (
```

Comment:
If you put that here, your string will contain a lot of leading white space. You could put it in the root of the module instead (below the imports).